share/etest/runners.sh

```diff
@@ -205,6 +216,15 @@ run_single_test()
     fi
 }
 
+# A wrapper function that calls suite_teardown if it is defined by user.
+__suite_teardown()
```

Comment:
Ah ha! I was wondering what this was when I saw you calling it up above. This is a nice solution @RaymondHuang210129 !